### PR TITLE
Buffs The Ectoscopic Sniffer

### PR DESCRIPTION
**Pull request by unit0016**
## About The Pull Request
Buffs the robotics ectoscopic sniffer to be much more obvious to spacemen's simple minds. 
I've increased the volume from 25 to 75 - a move aimed to make it so that everyone in robotics can hear it go off.
~~The visuals on blip have changed from green to red, to help draw the eye towards it.~~ Reverted pending different idea.
The examine text now explains how it functions in a more heavy-handed way.

## Why It's Good For The Game
Roboticists seem more interested in spacetiding and making big stompy mechs and modsuits instead of actually supporting the silicon players they were (Assumedly, considering the oldest record of either I can find is from 2012) made to support.
This should help draw their gaze back on objective without being /explicitly/ hostile to them.

## Changelog

:cl:
add: Ectoscopic sniffers are now louder, and explain their purpose on examine.
/:cl:

### CODE REVIEW

**Comment by Kubisopplay** [comment]
Bless you man

**Comment by zxaber** [comment]
The examine text pushes too far into the heavy-handed, if you ask me. Something something bikeshed comment. Otherwise, good idea.

To be honest, I've never had it (or perhaps noticed it) go off. I wonder if we could do something for ghosts like make it glow to their view.

**Comment by unit0016** [comment]
> The examine text pushes too far into the heavy-handed, if you ask me. Something something bikeshed comment. Otherwise, good idea.
> 
> To be honest, I've never had it (or perhaps noticed it) go off. I wonder if we could do something for ghosts like make it glow to their view.

I know, I was debating how exactly to phrase it for a bit honestly!
I wouldn't be opposed to reverting that part (or someone suggesting a better way to explain it.)

**Comment by Krysonism** [comment]
The volume change is good. 

I don't like the sprite change though, i think it looks better green.

**Comment by unit0016** [comment]
> I don't like the sprite change though, i think it looks better green.

It does, 200%. I just figured this was less intrusive than other possible solutions (Having it spit out a message, changing the sound to an alarm, etc etc.)

File: code/game/machinery/ecto_sniffer.dm [hunk 1]
```diff
@@ -73,6 +73,12 @@
 	ectoplasmic_residues = null
 	. = ..()
 
+/obj/machinery/ecto_sniffer/examine(mob/user)
+	. = ..()
+	. += span_notice("Any active ghost can leave a layer of ectoplasm on the ectoscopic sniffer, causing a small, audible blip. They may use this\
+	 to communicate that they wish to enter the shift as a positronic personality, to trigger a signaller assembly attached to the device, or a myraid\
```
Comment (interestingusernam3) [inline code/game/machinery/ecto_sniffer.dm:79]:
```suggestion
	 to communicate that they wish to enter the world as a positronic personality, to trigger a signaller assembly attached to the device, or a myraid\
```

Comment (unit0016) [reply code/game/machinery/ecto_sniffer.dm:79]: Good point.

**Comment by unit0016** [comment]
Considering revert the visual change as Kryson doesn't like it and he /actually/ knows how to sprite, honk.

BUT.
I'd like a way for it to still be obvious for deaf players.. does anyone have any ideas on how to do so?

**Comment by EricZilla** [comment]
Ectoplasmic mist? Like a lil spooky lines around it when it pings?

**Comment by unit0016** [comment]
> Ectoplasmic mist? Like a lil spooky lines around it when it pings?

That'd be neat. I'll revert the sprite change and play around with that idea.

**Comment by CameronWoof** [comment]
I showed up too late to ask why this object doesn't print a message to chat.

Going to do it anyway, though. Is there a reason this object doesn't print a message to chat?

**Pull request merged**